modules/ui/sections/raw_tag_editor.js

```diff
@@ -321,21 +321,21 @@ export function uiSectionRawTagEditor(id, context) {
     }
 
     function stringify(s) {
-        return JSON.stringify(s).slice(1, -1);   // without leading/trailing "
+        const stringified = JSON.stringify(s).slice(1, -1);   // without leading/trailing "
+        if (stringified !== s) {
+            return `"${stringified}"`;
+        } else {
+            return s;
+        }
     }
 
     function unstringify(s) {
-        var leading = '';
-        var trailing = '';
-        if (s.length < 1 || s.charAt(0) !== '"') {
-            leading = '"';
-        }
-        if (s.length < 2 || s.charAt(s.length - 1) !== '"' ||
-            (s.charAt(s.length - 1) === '"' && s.charAt(s.length - 2) === '\\')
-        ) {
-            trailing = '"';
+        const isQuoted = s.length > 1 && s.charAt(0) === '"' && s.charAt(s.length - 1) === '"';
+        if (isQuoted) {
+            return JSON.parse(`"${s}"`);
```

Comment:
Should we catch an exception here if the JSON string ends up being invalid JSON? Or is this guaranteed to be working with valid JSON?

Reply:
Good point. There could still be cases where a user enters an invalid quoted string, say something like `key="boo"far"`. Catching the exception here would allow us to only skip that particular offending tag, and still set the remaining tags properly. Which I think is better than the current situation where the full input field is ignored like reported in #10369